### ISSUE TITLE
Unable to /home or /spawn

### ISSUE THREAD

**Issue by ChiboYen** [Zrips/CMI#4819]
**Description of issue:**
Other plugins (worldguard) sometime cancel the teleportation event of CMI. The issue is CMI's fault because it doesn't implement spawn and home properly (while essentials never has this problem)
---

Cmi Version (using/cmi version): 8.7.12.4

Server Type (Spigot/Paperspigot/etc): Paper

Server Version (using /ver): 1.16.4

**Comment by Zrips** [comment]
That doesn't explain much of actual issue here. Does worldguard stops player teleportation? If it does, why? Is there some flags set to prevent this from happening? Is this only an issue with worldguard regions? Does this apply to global region or specific ones? I need more information on this issue to resolve it.

**Comment by ChiboYen** [comment]
No, worldguard would not cancel anything since it is not in a region (and my regions allow all commands). This issue appears to be related because CMI claims another plugin is cancelling the teleportation event, but it would appear that the bug resides within CMI itself as the other plugins (worldguard) cannot cancel the event, and since the issue is present even outside of regions.

**Comment by ChiboYen** [comment]
@Zrips It happened again, basically the issue has to do with #4880 somehow, let me know if I can provide any more info or data.
